src/content/docs/en/guides/cms/hygraph.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Hygraph & Astro
+description: Add content to your Astro project using Hygraph as a CMS
+type: cms
+service: Hygraph
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+
+[Hygraph](https://hygraph.com/) is a federated content management platform. It uses a simple GraphQL endpoint for fast distribution of content.
+
+## Integrating with Astro
+
+In this section, you'll create a [Hygraph](https://hygraph.com/) GraphQL endpoint to fetch with Astro.
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **A Hygraph account and a Hygraph project**. If you don't have an account, you can [sign up](https://app.hygraph.com/signup) for a free account and create a new Hygraph project. You can also use an existing project if you have one. 
+
+2. **Hygraph access permissions** - You can configure public Content API access permissions for requests from any website in **Project Settings > API Access**. If you don't have any permission setted, it's gonna ask you to **initialize some defaults** click on the **Yes, initialize defaults** button. Now you can use the different GraphQL endpoints
+
+    - **Content API** - For read and write operations. 
+    - **High Performance Read-only Content API** - For low latency queries, but no mutations.
+    - **Asset Upload API** - For asset uploading from your filesystem. For this to work, you'll a **Permanent Auth Token** with **Mutations**.
+    - **Management API** - This API handles all structural elements of a project, which can be utilized through the [Management SDK](https://hygraph.com/features/developer-operations/management-sdk).
+    
+ In this guide we are gonna work with the **High Performance Read-only Content API**
```

Comment:
As we're only using the high-performance content api, we can remove references to the rest.
```suggestion
```